docker/basic/Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM pytorch/pytorch:1.5.1-cuda10.1-cudnn7-devel
+
+#install git
+RUN apt-get update && apt-get install -y --no-install-recommends \
+         git && \
+     rm -rf /var/lib/apt/lists/*
+
+#setup opencv
+RUN echo '#!/bin/bash\n\
```

Comment:
There is no opencv in `pytorch/pytorch:1.5.1-cuda10.1-cudnn7-devel`, so we can just install the libs and set the timezone without checking `python -c "import cv2"`.